x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/bucket_span_estimator/estimate_bucket_span.ts

```diff
@@ -38,10 +38,10 @@ export function useEstimateBucketSpan() {
       end: jobCreator.end,
     },
     fields: jobCreator.fields.map((f) => (f.id === EVENT_RATE_FIELD_ID ? null : f.id)),
-    index: mlContext.currentDataView.title,
+    index: mlContext.selectedDataView.title,
```

Comment:
looks like I missed it in my PR for route resolvers 😬 could you please rename `mlContext` to `dataSourceContext`?

Reply:
Renamed them here [`7d120b9` (#159625)](https://github.com/elastic/kibana/pull/159625/commits/7d120b95964bbd3237adb6a106e47f34a81bab6b)